docs/7 Frontend/1 Overview.md

```diff
@@ -0,0 +1,19 @@
+# Frontend Overview
+
+When scaffolding a blockchain, Starport also creates a Vue frontend application in `vue` directory. If this directory exists, `starport serve` automatically runs `npm run serve`.
+
+The frontend app uses packages from [`tendermint/vue`](https://github.com/tendermint/vue).
```

Comment:
Should we explicitly name the packages? i.e. something like:

The front-end app is built using the @starport/vue and @starport/vuex packages. More information and documentation can be found at the [`tendermint/vue`](https://github.com/tendermint/vue) monorepo.

Reply:
Sounds good, thanks!